src/test/java/com/mongodb/hibernate/jdbc/MongoConnectionTests.java

```diff
@@ -79,259 +92,320 @@ void testClosedWhenSessionClosingThrowsException() {
             // then
             assertTrue(mongoConnection.isClosed());
         }
-    }
-
-    @Nested
-    class TransactionTests {
 
         @Nested
-        class SetAutoCommitTests {
+        class TransactionTests {
```

Comment:
Why were `MongoConnectionTests.TransactionTests` moved to `MongoConnectionTests.CloseTests.TransactionTests`, and `GetMetaDataTests` created in `CloseTests`? Neither seems right.

Reply:
You are right. I might have screwed up during syncing up with latest main branch.